src/main/java/org/cactoos/text/LengthOfText.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Scalar;
+import org.cactoos.Text;
+
+/**
+ * The length of {@link Text}.
+ * The length is equal to the number of Unicode code units in the text.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Vseslav Sekorin (vssekorin@gmail.com)
+ * @version $Id$
+ * @since 0.11
+ */
+public final class LengthOfText implements Scalar<Integer> {
+
+    /**
+     * The origin text.
+     */
+    private final Text origin;
+
+    /**
+     * Ctor.
+     * @param string The string
+     */
+    public LengthOfText(final String string) {
+        this(new StringAsText(string));
+    }
+
+    /**
+     * Ctor.
+     * @param text The text
+     */
+    public LengthOfText(final Text text) {
+        this.origin = text;
+    }
+
+    @Override
+    public Integer value() throws Exception {
+        return this.origin.asString().length();
```

Comment:
Explain a little more in detail what is the problem with the exception in this class, please.